JS/ui.js

```diff
@@ -50,11 +69,24 @@ class UI {
     const booksUL = document.getElementById('books');
     const newBookToAppend = document.createElement('li');
 
+    //   <div>
+    //   <h3>"${book.title}" </h3>
+    //   <span> by ${book.author}</span>
+    // </div>
+    // <button
+    //   data-id="${book.id}"
+    //   type="button">Remove
+    // </button>
```

Comment:
- [ ] Please do not leave commented pieces of code in your codebase as they make your project look messy and unprofessional. You can use git history if you need to review a previous version of your project.

Reply:
Thank you for your observation. I have just removed the comment